ansible_image_validation/validate-vm-images.yaml

```diff
@@ -10,7 +10,8 @@
   remote_user: "{{ lookup('env', 'ADMIN_USER_NAME') }}"
   become: true
   vars:
-    ansible_become_pass: "{{ lookup('env', 'ADMIN_PASSWORD') }}"
+    ansible_become_pass: "{{ lookup('env', 'ADMIN_USER_NAME') }}"
```

Comment:
Why is password variable lookup is from ADMIN_USER_NAME?

Reply:
thanks for correcting, changed it to ADMIN_PASSWORD